wordpress-mobile/WordPress-iOS
# Activity Log v0.001 with no storage

**Another step for** #7832 

This PR provides a simple fetch for a site last 100 activities and displays their summary, actor, timestamp and a varying background color: light blue for rewindable activities, white for non rewindable ones. In the future we will, instead of changing the background, add the Rewind button accordingly.
It is a very crude version 0.001 in anticipation to adding a store according to the new Flux pattern and support for Rewind.

**To test:**
Go to a Jetpack site -> Activity logged in as an admin for the site
Check that what you see matches the web

Needs review: @koke
